### ISSUE TITLE
Riemann executable doesn't work

### ISSUE THREAD

**Issue by chris-martin** [NixOS/nixpkgs#21790]
I think #20821 was closed prematurely; 7bcc6e9c45afd9a7f69caa2ad6e75fb0051eab41 added the riemann run script, but it seems to need a wrapper to provide `java`, among other things.

```
$ nix-shell -p riemann --run riemann --pure
/nix/store/2l3ickwgfp8521b5npkfq2468rd8f02z-riemann-0.2.9/bin/riemann: line 50: exec: java: not found
$ nix-shell -p riemann -p jre --run riemann --pure
Error: Could not find or load main class riemann.bin
```

**Comment by FRidh** [comment]
cc @pSub

**Comment by globin** [comment]
Fixed by @fpletz in 34eae3f73ae076bf17ae6091763891939b303596
